### PR TITLE
Updated the content of the confimration page

### PR DESCRIPTION
**Pull request by BroiSatse**
### Context

[Ticket: https://dfedigital.atlassian.net/browse/CPDNPQ-000
](https://dfedigital.atlassian.net/browse/CPDNPQ-992)

### Changes proposed in this pull request

Simple update of some content on confirmation page as per new designs

### CODE REVIEW

**Review by javier-npq — APPROVED** [review]

File: config/locales/en.yml [hunk 1]
```diff
@@ -2,6 +2,19 @@ en:
   time:
     formats:
       admin: "%R on %d/%m/%Y"
+  courses:
+    alternative_names:
```
Comment (Lockyy) [inline config/locales/en.yml:6]: Would it be possible to change the key to identifier? To match this https://github.com/DFE-Digital/npq-registration/pull/766/files#diff-44438ce218f5287c58d0017f965d888715635d94280669896f75841fbd7b4cd7R236? 

It'd work into the consistent naming system that Emma and I are introducing through the app.

Comment (BroiSatse) [reply config/locales/en.yml:6]: I actually wondered whether this is something that should be part of the database model? It feels a bit odd to have a part of naming in the database and the rest in translations...

Comment (Lockyy) [reply config/locales/en.yml:6]: This is actually something we're specifically moving away from, the names being in the DB was causing several issues. Moving to just holding the identifier in the DB and then names in the locale file has made things easier. I've taken what you've said on board in that PR and swapped course.identifier with course.name.

Comment (Lockyy) [reply config/locales/en.yml:6]: The issues were down to the DB values causing inconsistencies across this and the ECF app and needing to refresh the DB to make changes. Having the identifier in the DB and the locales handling how that is converted into what the user sees makes deploying and managing things easier.

**Review by Lockyy — APPROVED** [review]

**Pull request merged**